certora/scripts/run-all.sh

```
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+certora/scripts/apply-patch.sh
+
+certoraRun certora/confs/folio_prerequisities.conf
+
+for FILE in certora/confs/properties/*.conf
+do
+    echo ${FILE}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)